test/webpack.config.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+const path = require('path')
+const webpack = require('webpack')
+const assert = require('assert')
+
+function webpackForTest(testFileName) {
+  const match = testFileName.match(/\/?([^\/]*)\.ts$/)
+  if (!match) {
+    assert(false, 'wrong filename:' + testFileName)
+  }
+
+  const test = {
+    mode: 'production',
+    cache: true,
+    externals: [
+      {
+        'xrpl-local': 'xrpl',
+        net: 'null',
+      },
+    ],
+    entry: testFileName,
+    output: {
+      library: match[1].replace(/-/g, '_'),
```

Comment:
What is match[0] here?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match

Reply:
It's a regex thing